necroHeal.lic

```diff
@@ -0,0 +1,43 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placement comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status

```

Comment:
Hey there, I think there's a disconnect between what Lich is and what Dependency is. Basically, your script is good and all, but Dependency is a scripting automation suite, it isn't the same thing as Lich. Dependency users will find value in it, so they will add it, but they also think that they have a use for it to run inside the Dependency engine. This is separate from your script being added, they just want it to be even more useful. 

Also, not sure if it's clear or not, but even though it is being used by safe-room as part of the suite, it can also be used in befores and afters as well as by using the ; invoker to run it stand-alone.